# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/moz.build
# Commit: 0e3bc029e59f
# Full Hash: 0e3bc029e59f5f0319a57c469d7c0991081379ff
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-10 21:43:01
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 8b2ffdf05455 -r 0e3bc029e59f dom/credentialmanagement/identity/moz.build
--- a/dom/credentialmanagement/identity/moz.build	Tue Jun 10 11:49:57 2025 +0000
+++ b/dom/credentialmanagement/identity/moz.build	Tue Jun 10 11:49:57 2025 +0000
@@ -13,16 +13,22 @@
     "IdentityCredentialSerializationHelpers.h",
     "IdentityNetworkHelpers.h",
     "NavigatorLogin.h",
+    "WebIdentityChild.h",
+    "WebIdentityHandler.h",
+    "WebIdentityParent.h",
 ]
 
 IPDL_SOURCES += [
-    "IPCIdentityCredential.ipdlh",
+    "PWebIdentity.ipdl",
 ]
 
 UNIFIED_SOURCES += [
     "IdentityCredential.cpp",
     "IdentityNetworkHelpers.cpp",
     "NavigatorLogin.cpp",
+    "WebIdentityChild.cpp",
+    "WebIdentityHandler.cpp",
+    "WebIdentityParent.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")